packages/fxa-auth-server/package.json

```diff
@@ -27,7 +27,9 @@
     "write-emails": "node -r ts-node/register ./scripts/write-emails-to-disk.js",
     "clean-up-old-ci-stripe-customers": "ts-node ./scripts/clean-up-old-ci-stripe-customers.js",
     "paypal-processor": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/paypal-processor.ts",
-    "subscription-reminders": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/subscription-reminders.ts"
+    "subscription-reminders": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/subscription-reminders.ts",
+    "write-templates": "ts-node lib/senders/emails/render-to-disk.ts",
+    "preview": "ts-node lib/senders/emails/preview-in-maildev.ts"
```

Comment:
Since this command exists in the fxa-auth-server space, calling it "preview" isn't super clear. Maybe `maildev-preview`?